ci/k8s/deployment.yml

```diff
@@ -180,6 +180,25 @@ spec:
             cpu: "10m"
           limits:
             cpu: "100m"
+      - name: cloudsql-proxy
+        image: gcr.io/cloudsql-docker/gce-proxy:1.14
```

Comment:
Change to EU repo and most recent version `eu.gcr.io/cloudsql-docker/gce-proxy:1.16`